### ISSUE TITLE
[WIN 11]1.8.1 Joysticks messed up after installation

### ISSUE THREAD

**Issue by Reupireup** [jellyfin/jellyfin-media-player#374]
**Describe the bug**
After installing Jellyfin 1.8.1 (over 1.8.0 release), my joysticks (generic x-box like joysticks) are not usable, analog inputs are considered like digital 0-1 (instead of 0-255 values), some axis are mapped with buttons.
After reading release notes, I think it comes from :
> Add input map for DualShock 4 connected via USB (https://github.com/jellyfin/jellyfin-media-player/pull/359)

**To Reproduce**
on windows 11, install 1.8.1, reboot (as the installation wizard suggest), test joystick with x360ce (or via the parameter pannel > joysticks)

**Desktop (please complete the following information):**
 - OS: WIN 11 
 - Version 1.8.1
 - Installation Method windows installer

**Additional context**
after uninstall 1.8.1 the problem is still present, can't play game with my joysticks anymore :'(

**Comment by iwalton3** [comment]
I'm not sure how an input map configuration could break joysticks specifically. Are you sure the application isn't running? Maybe steam could be remapping inputs somehow?

**Comment by Reupireup** [comment]
I'm pretty sure it cames from the 1.8.1 update, I played video games (trackmania), then update jellyfin client, restart windows as wizard request it.
During the reboot phase, windows has been updated cause I saw the message with the wheel for ~10 seconds.
Just after the reboot I launched trackmania and saw the joystick problems (jellyfin player was not running).
Then, I uninstalled jellyfin player to see if the configuration comes back, with no effect.
